production/helm/loki-stack/values.yaml

```diff
@@ -41,7 +38,6 @@ filebeat:
 
 logstash:
   enabled: false
-  fullnameOverride: logstash-loki
```

Comment:
The problem is this is required for the default chart to work with logstash.

Unfortunately there is no way to know the service name within the value file.  This is used by both the filebeat.yml and the main output of logstash.

Do you have an alternative ?

Reply:
I don't think it makes sense to break existing deployments for a disabled by default feature. A better alternative is to enhance the upstream chart to support templating of the necessary values so you can specify templated values in the values.yaml. For example, here is the config from my values.yaml to configure Grafana to use Loki:
```yaml
  grafana:
    additionalDataSources:
      - name: Loki
        type: loki
        access: proxy
        url: http://{{ .Release.Name }}-loki:3100
        jsonData:
          maxLines: 500
```
See the templating [PR](https://github.com/helm/charts/pull/22123) I did for Grafana's ingress annotations to see how it's done. I don't use logstash or know which fields have to be templated to be able to help with this. 

Or if that's not feasible then simply documenting how users can adjust the chart for use with Logstash should suffice.

Reply:
Gonna need some time to figure this out. May be we can add those override in the docs via —set for now as examples for logstash.

Reply:
I found an other bug with those templates, for example promtail in the umbrella chart will use the stack template but with value from the subchart and the template is wrong. 

Subchart and templates is not a fun area.

Reply:
Alright I've sent a PR to make this part of the doc instead.